lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +19,94 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas: 
+    "list-text actions"
+    "list-sub-text actions";
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
   &:first-of-type {
-    border-top: $nx-border;
+    border-top: 1px solid #d1d6f0;
   }
+}
 
-  &.nx-error {
-    border: $nx-border-error;
-    color: $nx-text-color-error;
-  }
+.nx-list__text {
+  @include semi-bold();
 
-  > .fa, > .svg-inline--fa {
-    margin-right: $nx-spacing-md;
-  }
+  grid-area: list-text;
 }
 
-.nx-list--bulleted {
-  .nx-list__item {
-    @include regular();
+.nx-list__subtext {
+  @include regular();
 
-    border: none;
-    list-style: disc outside;
-    margin: $nx-spacing-sm 10px $nx-spacing-sm 36px;
-    overflow: visible;
-    padding: 0;
-    white-space: normal;
-  }
+  display: block;
+  grid-area: list-sub-text;
+  // top margin is measured down from the top of the objects grid row
+  margin: $nx-spacing-md 0 0 0;
 
-  .nx-list__item {
-    list-style-type: disc;
+  &.nx-truncate-ellipsis {
+    white-space: nowrap;
+  }
+}
 
-    .nx-list__item {
-      list-style-type: circle;
+.nx-list__item--empty {
+  display: list-item;
+  text-align: center;
 
-      .nx-list__item {
-        list-style-type: square;
-      }
-    }
+  .nx-list__text {
+    @include regular();
   }
 }
 
-.nx-list__item--empty {
-  @include regular();
+.nx-list__item--error {
```

Comment:
Zeplin has the error alert going all the way across; no left and right padding.

Reply:
> Zeplin has the error alert going all the way across; no left and right padding.

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/75155c56c362ba2b6095ad9c29a542c26cc62786